### ISSUE TITLE
Feeds surface content older than 6 months (based on createdAt date)

### ISSUE THREAD

**Issue by veekaybee** [bluesky-social/social-app#6197]
### Steps to Reproduce

1. If you have the [MLSky feed](https://bsky.app/profile/alexr.bsky.social/feed/MLSky), refresh the feed on any platform. 
2.  You can see content that's 6 months old and from 2018, likely via Twitter imports in the attached images. [In one of the cases](https://atproto-browser.vercel.app/at/did:plc:jf4udtyuylqocyyrqdwqoxuf/app.bsky.feed.post/3lahy6mbs5s22), it's because the `createdAt` date on the plc record is old, and likely the source of order of ingest in the stream.  
3. [Other folks have noticed](https://bsky.app/profile/b0rk.jvns.ca/post/3la4parah7i2x) this issue with other feeds as well, particularly as people import more of their past Tweets when just onboarding onto the platform. 

Given that many feeds are user-generated, it may make sense on the AppView side to limit feed content to < 7 days with clear labeling of intent and date limit, although with specific feeds like "📌" this might not make as much sense. 

In general, based on users' previous experiences with social network "feed" speed, it feels like recency could be enforced on the AppView client side.  

### Attachments

<img width="557" alt="Screenshot 2024-11-09 at 9 16 00 PM" src="https://github.com/user-attachments/assets/231c3a01-9af8-489c-94ef-72162bc89a8c">
e 
<img width="703" alt="Screenshot 2024-11-09 at 9 07 43 PM" src="https://github.com/user-attachments/assets/4b5ecf5c-39e2-4cd4-8d58-12ab2bf405a9">

### What platform(s) does this occur on?

Web (Desktop)

### Device Info

Web/iOS but not specific to either

### What version of the app are you using?

Build version: 1.93.0; Bundle info: 7731136 (prod); Bundle date: 24110818; Platform: web

**Comment by mary-ext** [comment]
This is on custom feeds, and the thing about feeds is they don't have to be reverse-chronological, so it's pretty much the feeds' responsibility to make sure that the posts it's returning are reverse-chron.

**Comment by mozzius** [comment]
Yeah gonna close with this one, custom feeds can serve whatever posts they like
